docs/mesos/UsageGuideDockerContainerizer.md

```diff
@@ -0,0 +1,178 @@
+<!--- master only -->
+> ![warning](../images/warning.png) This document applies to the HEAD of the calico-containers source tree.
+>
+> View the calico-containers documentation for the latest release [here](https://github.com/projectcalico/calico-containers/blob/v0.18.0/README.md).
+<!--- else
+> You are viewing the calico-containers documentation for release **release**.
+<!--- end of master only -->
+
+# Calico-Mesos Usage Guide with the Docker Containerizer
+
+The following instructions outline the steps required to enable and manage 
+Calico networked containers launched from Marathon using the Mesos Docker
+containerizer.  This guide covers:
+-  Setting up Docker multi-host networking
+-  Installing and configuring Calico
+-  Creating a Docker network and managing network policy
+-  Launching a container
+
+## Prerequisites
+
+You can easily configure a mesos cluster by running through
+the [Calico Mesos Vagrant guide](./Vagrant.md). If you do this, you can skip
+directly to [Launching Containers](#launching-containers).
+
+To utilize the Docker libnetwork plugin feature of Docker, it is necessary to 
+run with a Docker version 1.9 or higher, and to configure Docker to use a
+cluster store.  
+
+#### Required Machines 
+
+The cluster must contain the following machines:
+
+- Mesos Master
+- Mesos Agent(s)
+	- Docker 1.9+ (see below)
+
+#### Required Services
+
+You will also need to run the following services somewhere in your cluster:
+
+- Marathon
+- Etcd
+
+It is easiest to just run these services on the Mesos Master machine.
+Each machine in the cluster must have access to these services.
+
+#### Set up Docker daemon to use etcd as a cluster store
```

Comment:
I think this gets moved to be a prereq in the Install Calico with Docker Containerizer guide now